src/main/deploy/pathplanner/paths/New Path.path

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Do we need this file or was this from when you were trying to figure out the points to use?